src/Components/Timer/Timer.js

```diff
@@ -4,87 +4,87 @@ import style from './Timer.module.scss';
-                lastUnit='m'
-                startImmediately={false}
-                timeToUpdate={10}
-                checkpoints={[
-                    {
-                        time: 0,
-                        callback: () => timerDone(),
-                    },
-                ]}
-            >
-                {({ start, pause, reset }) => (
-                    <>
-                        <div className={style.baseTimer}>
-                            <svg className={style.baseTimerSvg} viewBox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'>
-                                <g className={style.baseTimerCircle}>
-                                    <circle className={style.baseTimerPathElapsed} cx='50' cy='50' r='45' />
-                                    <path
-                                        id='base-timer-path-remaining'
-                                        strokeDasharray='283'
-                                        className={style.baseTimerPathRemaining}
-                                        d="
-                                            M 50, 50
-                                            m -45, 0
-                                            a 45,45 0 1,0 90,0
-                                            a 45,45 0 1,0 -90,0
-                                        "
-                                    />
-                                </g>
-                            </svg>
-                            <p className={style.baseTimerLabel}>
-                                {<Timer.Minutes />}:{<Timer.Seconds 
-                                    formatValue={(value) => `${(value < 10 ? `0${value}` : value)}`}
-                                />}
-                            </p>
-                        </div>
-                        <div className={style.timerControls}>
-                            <button className={style.timerControlButton} onClick={() => {
-                                start();
-                                // setCircleDashArray();
-                            }}>
-                                Start
-                            </button>
-                            <button className={style.timerControlButton} onClick={pause}>Pause</button>
-                            <button className={style.timerControlButton} onClick={reset}>Reset</button>
-                        </div>
-                    </>
-                )}
-            </Timer>
-        </div>
-    )
+  return (
+    <div className={style.countdownTimer}>
+      <Timer
+        direction='backward'
+        initialTime={settings.workInterval * 60000}
+        lastUnit='m'
+        startImmediately={false}
+        timeToUpdate={10}
+        checkpoints={[
+          {
+            time: 0,
+            callback: () => timerDone(),
+          }, 
+          /* These colors can be found in our variables.scss file (salmon progression) */
+          {
+            time: settings.workInterval * 45000,
```

Comment:
Love this color changing functionality, so slick!

Reply:
Yes! Great idea @Abdeboskey to toggle these on quarter intervals.

@rwilliams659 I'm glad we could all take a look at it together as well! Naturally, I forgot to co-author these commits...

Reply:
Haha no worries on the co commits. I'm stoked that you were able to get this working, thanks for being down to spend time on it together!